staging/src/k8s.io/kube-aggregator/pkg/apis/apiregistration/v1/helper/helpers_test.go

```diff
@@ -0,0 +1,118 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import (
+	"k8s.io/kube-aggregator/pkg/apis/apiregistration/v1"
+	"reflect"
+	"testing"
+)
+
+var (
+	a v1.APIServiceConditionType = "A"
+	b v1.APIServiceConditionType = "B"
+	c v1.APIServiceConditionType = "C"
+)
+
```

Comment:
looks that you removed two tests: TestIsAPIServiceConditionTrue and TestSetAPIServiceCondition?

Reply:
good point